doc/Requirements Specification/Parts/Terms and Definitions.lyx

```diff
@@ -1204,6 +1249,50 @@ newglossaryentry{Kieker}{
 }
 \end_layout
 
+\begin_layout Plain Layout
+
+\end_layout
+
+\begin_layout Plain Layout
+
+
+\backslash
+newglossaryentry{measurement software}{
```

Comment:
but it isn't in the PDF, is it?

if it is not, probably it is not used yet. 
So everything is fine.